eng/code-quality-reports/src/main/resources/checkstyle/checkstyle.xml

```diff
@@ -267,7 +267,9 @@ page at http://checkstyle.sourceforge.net/config.html -->
         -->
         <module name="com.azure.tools.checkstyle.checks.ServiceClientChecks"/>
 
-        <!--Fluent method should not start with 'with' -->
-        <module name="com.azure.tools.checkstyle.checks.FluentMethodNameCheck"/>
+        <!-- Fluent method should not start with 'with' -->
+        <module name="com.azure.tools.checkstyle.checks.FluentMethodNameCheck">
```

Comment:
I think we should also add the words "set" to this.

Reply:
misunderstood. will update string array to set

Reply:
I meant, as a default, we should have the keywords, "with" and "set".
```xml
<property name="avoidStartWords" value="with, set"/>
```